src/pages/home/report/ReportActionItem.tsx

```diff
@@ -342,6 +343,8 @@ function ReportActionItem({
                 return;
             }
 
+            ReportActionComposeFocusManager.composerRef.current?.blur();
+            ReportActionComposeFocusManager.editComposerRef.current?.blur();
```

Comment:
Instead of calling individual `blur`, can we define a utility function in `ReportActionComposeFocusManager` and call `blur` when focussed? I think that would be neater.